tests/publiccloud/create_aistack_env.pm

```diff
@@ -116,12 +121,11 @@ sub install_aistack_chart {
     my $gitlab_clone_url = 'https://git:' . $git_token . '@' . $repo_url;
     assert_script_run("git clone $gitlab_clone_url");
     assert_script_run("curl " . data_url("aistack/$vf_name") . " -o $vf_name", 60);
-    assert_script_run("curl -o $vf_name $ai_chart_repo", timeout => 120);
 
     # local-path-storage.yaml is a copy off https://raw.githubusercontent.com/rancher/local-path-provisioner/v0.0.28/deploy/local-path-storage.yaml
     assert_script_run("curl " . data_url("aistack/$local_storage_name") . " -o $local_storage_name", 60);
     assert_script_run("kubectl apply -f $local_storage_name", timeout => 120);
-    assert_script_run("helm upgrade --install suse-private-ai private-ai-charts --namespace $namespace --create-namespace --values $vf_name --set open-webui.ingress.class=nginx", timeout => 600);
+    assert_script_run("helm upgrade --install suse-private-ai private-ai-charts --namespace $namespace --create-namespace --values $vf_name --set open-webui.ingress.class=nginx", timeout => 800);
     assert_script_run("kubectl get all --namespace $namespace");
     sleep 180;
```

Comment:
In general a generic sleep should be avoided: better to run a last status check that expected output is ok.

Reply:
fixed to handle in the loop and reduce the sleep wait